profiles/README.MD

```diff
@@ -0,0 +1,7 @@
+
+This is the main repo page for published JSON-LD profiles for xAPI. 
```

Comment:
I'm setting up a separate repo. I don't expect each group to have their own repo. Of course we would allow that, but we also want to be able to curate profiles. I'm going to close this PR and fix these issues and submit a new PR. Thanks for the feedback on everything.